README.md

```diff
@@ -41,36 +41,7 @@
 
 Parse Server works with the Express web application framework. It can be added to existing web applications, or run by itself.
 
-The full documentation for Parse Server is available in the [wiki](https://github.com/parse-community/parse-server/wiki). The [Parse Server guide](http://docs.parseplatform.org/parse-server/guide/) is a good place to get started. An [API reference](http://parseplatform.org/parse-server/api/) and [Cloud Code guide](https://docs.parseplatform.org/cloudcode/guide/) are also available. If you're interested in developing for Parse Server, the [Development guide](http://docs.parseplatform.org/parse-server/guide/#development-guide) will help you get set up.
-
-- [Getting Started](#getting-started)
-    - [Running Parse Server](#running-parse-server)
-        - [Locally](#locally)
-        - [Docker](#inside-a-docker-container)
-        - [Saving an Object](#saving-your-first-object)
-        - [Connect an SDK](#connect-your-app-to-parse-server)
-    - [Running elsewhere](#running-parse-server-elsewhere)
-        - [Sample Application](#parse-server-sample-application)
-        - [Parse Server + Express](#parse-server--express)
-    - [Configuration](#configuration)
-        - [Basic Options](#basic-options)
-        - [Client Key Options](#client-key-options)
-        - [Email Verification & Password Reset](#email-verification-and-password-reset)
-        - [Custom Pages](#custom-pages)
-        - [Using Environment Variables](#using-environment-variables-to-configure-parse-server)
-        - [Available Adapters](#available-adapters)
-        - [Configuring File Adapters](#configuring-file-adapters)
-        - [Logging](#logging)
-- [Live Queries](#live-queries)
-- [GraphQL](#graphql)
-- [Upgrading to 3.0.0](#upgrading-to-300)
-- [Support](#support)
-- [Ride the Bleeding Edge](#want-to-ride-the-bleeding-edge)
-- [Contributing](#contributing)
-- [Contributors](#contributors)
-- [Sponsors](#sponsors)
-- [Backers](#backers)
-
+The full documentation for Parse Server is available [here](https://docs.parseplatform.org/parse-server/guide/). An [API reference](http://parseplatform.org/parse-server/api/) and [Cloud Code guide](https://docs.parseplatform.org/cloudcode/guide/) are also available.
```

Comment:
May be we can have a detailed redirection system like

```md
Documentation:
- [Home](https://parseplatform.org/)
- [Get started](https://docs.parseplatform.org/parse-server/guide/)
- [GraphQL](https://docs.parseplatform.org/graphql/guide/)
- etc...
```

Reply:
I like the suggestion. I'll make this change once the docs site is finalised!